Projects/Team2/derbackAPP/SeleniumTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Threading;
+
+namespace SeleniumTest
+{
+    class Program
+    {
+        public IWebDriver driver;
+
+        [OneTimeSetUp]
+        public void StartUp()
+        {
+            driver = new ChromeDriver();
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+        }
+        [OneTimeTearDown]
+        public void TestEnd()
+        {
+            driver.Quit();
+        }
+        [Test]
+        public void TestPubg()
+        {
+            string link = "https://lite.pubg.com/ru/news/";
+            driver.Navigate().GoToUrl("https://lite.pubg.com/ru/download/");
+            driver.Navigate().GoToUrl("https://lite.pubg.com/ru/news/");
+            Assert.AreEqual(link, driver.Url);
+            Thread.Sleep(2000);
```

Comment:
try to avoid Thread.Sleep
it is better just to increase ImplicitWait

Reply:
many threadsleep through my bad internet